### PR TITLE
Fix tests by using pinned jenkins image

### PR DESCRIPTION
**Pull request by janisz**
Jenkins LTS image was updated recently and it no longer violates policy.
This PR switch from LTS to 2.77 to ensure policy is violated.

### CODE REVIEW

**Review by dhaus67 — APPROVED** [review]

**Comment by janisz** [comment]
**Graphite Merge Job**

Current status: 🔄 Waiting for checks to pass

This pull request is currently waiting for CI and mergeability checks to pass before being merged.

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/stackrox/jenkins-plugin/176?utm_source=stack-comment).


Job Reference: k1TsZxd9Ccz9YmPqRkDw

**Comment by janisz** [comment]
**Graphite Merge Job**

Current status: 🔄 Waiting for checks to pass

This pull request is currently waiting for CI and mergeability checks to pass before being merged.

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/stackrox/jenkins-plugin/176?utm_source=stack-comment).


Job Reference: kZf9XRqdqG9eC6V9x71x

**Pull request merged**
